xml/System.Security.Permissions/CodeAccessSecurityAttribute.xml

```diff
@@ -79,7 +79,10 @@
     <remarks>
       <format type="text/markdown"><![CDATA[  
   
-## Remarks  
+## Remarks
+
+[!INCLUDE[cas-deprecated](~/includes/cas-deprecated.md)]
+
 > [!IMPORTANT]
 > Partially trusted code is no longer supported. This attribute has no effect in .NET Core.
```

Comment:
This alert and similar instances of "no longer supported" appear in a lot of places, but they seem to be redundant after the new include and could be deleted.